### PR TITLE
tembo ai launch

### PR DESCRIPTION
**Pull request by ChuckHend**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

**Comment by FloorD** [comment]
I don't think you need me as a co-author here.

**Review by FloorD — APPROVED** [review]

**Comment by ChuckHend** [comment]
@FloorD do we have a social image for this one?

**Review by DarrenBaldwin07 — APPROVED** [review]

File: src/content/blog/2024-08-16-tembo-ai/index.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Tembo already supports hosting hundreds of [embedding models](https://tembo.io/docs/product/cloud/apps/embeddings), including those privately trained on CPU isolated to your Tembo Cloud environment. Tembo AI brings GPU support into the platform, providing significantly improved latency and throughput. With Tembo AI, users have all the same flexibility as our embedding service and now have the ability to call chat models on their Tembo infrastructure.
+
+
+### **Beyond third-party hosting providers**
+
+Hooking into third party hosting providers is not enough for many users. For users who do not want to share their data with LLM providers, there is only one-option: self-hosted. [Tembo Self Hosted](https://tembo.io/docs/product/software/tembo-self-hosted/overview) users get embedding and chat models built into their solution, allowing them to generate embeddings and call chat models without data leaving their infrastructure. Tembo Cloud and Tembo Self Hosted users have embedding and chat models available to them at and hook into their database with minimal configuration.
+
+
+#### **Seamless Integration with Tembo Cloud and Self-Hosted Options**
```
Comment (samay-sharma) [inline src/content/blog/2024-08-16-tembo-ai/index.mdx:25]: We should have a paragraph either here or at the top saying something like "Building AI apps is complicated. Users have to have a database with their data, a vector store, a model provider, and glue to tie all of them together. Our goal with Tembo AI is to give users a one stop shop to get their AI applications off the ground as fast as possible.

You have your dataset, vector search, model hosting and calling of models all integrated in a simple solution. That way you can get started with your app and push it to production using a single simple solution.

Comment (samay-sharma) [reply src/content/blog/2024-08-16-tembo-ai/index.mdx:25]: I feel this context is the only thing missing.

**Review by samay-sharma — COMMENTED** [review]
I have one major point of feedback which is explaining a bit on why we think this makes developers lives easier.

Once that's addressed, I think this is good to ship.

File: src/content/blog/2024-08-16-tembo-ai/index.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+date: 2024-08-16T09:00
+planetPostgres: false
+---
+
+The Tembo AI platform is designed specifically to provide engineering teams with the most convenient and secure way to build with artificial intelligence and PostgreSQL. Tembo AI is a comprehensive solution that integrates both chat and embedding models into your PostgreSQL environment, enabling a seamless and powerful experience for building GenAI applications.
+
+
+### **GPU Compute in Tembo Cloud and Self Hosted**
+
+Tembo already supports hosting hundreds of [embedding models](https://tembo.io/docs/product/cloud/apps/embeddings), including those privately trained on CPU isolated to your Tembo Cloud environment. Tembo AI brings GPU support into the platform, providing significantly improved latency and throughput. With Tembo AI, users have all the same flexibility as our embedding service and now have the ability to call chat models on their Tembo infrastructure.
```
[the remark this answers is not in the document]
Comment (ChuckHend) [reply src/content/blog/2024-08-16-tembo-ai/index.mdx]: https://github.com/tembo-io/website/pull/610/commits/936fd36d64c154e012f6b12b221a2468db910c3f

**Review by samay-sharma — APPROVED** [review]
@FloorD , @ChuckHend, this is good to go. Please update the date, add a header image and ship it.

**Pull request merged**